src/plugins/data/common/search/search_source/legacy/default_search_strategy.test.ts

```diff
@@ -17,51 +17,50 @@
  * under the License.
  */
 
-import { HttpStart } from 'src/core/public';
-import { coreMock } from '../../../../../core/public/mocks';
-import { getCallMsearch } from './call_msearch';
 import { defaultSearchStrategy } from './default_search_strategy';
 import { LegacyFetchHandlers, SearchStrategySearchParams } from './types';
 import { BehaviorSubject } from 'rxjs';
 
 const { search } = defaultSearchStrategy;
 
-const msearchMock = jest.fn().mockResolvedValue({ body: { responses: [] } });
-
-describe('defaultSearchStrategy', function () {
-  describe('search', function () {
+describe('defaultSearchStrategy', () => {
+  describe('search', () => {
```

Comment:
In this test file, I needed to move a few things around because it was testing `callMsearch`, which lives in `public`. So I instead mocked `callMsearch` and simply tested that the function was called with the correct args.

Then I moved the existing logic asserting that `http.post` was called correctly into a separate unit test which lives alongside `callMsearch`.